framework/src/interfaces/TaggingInterface.C

```diff
@@ -224,13 +270,19 @@ TaggingInterface::accumulateTaggedLocalResidual()
 {
   for (auto & re : _re_blocks)
     *re += _local_re;
+  for (auto & absre : _absre_blocks)
+    for (const auto i : index_range(_local_re))
+      (*absre)(i) += std::abs(_local_re(i)) + std::numeric_limits<Number>::min();
```

Comment:
Since you're adding the small value for the purpose of avoiding a divide by zero in ReferenceResidualProblem, it seems like it would make more sense to add there right where you're going to do that division, rather than here, because this could get used for other things.

Reply:
Yeah not a bad idea. I assume there's an easy way to do that with `NumericVector<Number>` rather than element by element?

Reply:
this was addressed in the vein @bwspenc suggested

Reply:
(although I do note that you'll be screwed if any element in the vector == `-std::numeric_limits<Number>::min()` 😛